src/components/BooleanPill/BooleanPill.props.ts

```diff
@@ -0,0 +1,13 @@
+import { BooleanPillConfig } from './BooleanPill.styles';
+import { AvatarAppearance } from '../Avatar/types';
+
+export type BooleanPillProps = {
+  text: string;
+  state?: 'default' | 'disabled' | 'selected';
```

Comment:
`selected` should be as `boolean` props `isSelected`, same as `isInverted`.
In `state` props are states which cant be mixed with other one, we can set `active` and `disabled` in the same time ;)